includes/class-wc-stripe-order-helper.php

```diff
@@ -0,0 +1,613 @@
+<?php
+
+use Automattic\WooCommerce\Enums\OrderStatus;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Stripe_Order_Helper class.
+ */
+class WC_Stripe_Order_Helper {
+	/**
+	 * Meta key for Stripe currency.
+	 *
+	 * @string
```

Comment:
```suggestion
	 * @var string
```

Same suggestion for the other contsants.

Reply:
@wjrosa  the suggestion is not applied to the other constants.

Reply:
Sorry! Updated them all in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4613/commits/ba27b2bbae75949b066d8b538edfc5dac6d28cf3